src/libraries/System.Data.OleDb/src/OleDbComWrappers.cs

```diff
@@ -37,8 +37,7 @@ protected override object CreateObject(IntPtr externalComObject, CreateObjectFla
         {
             Debug.Assert(flags == CreateObjectFlags.UniqueInstance);
 
-            Guid errorInfoIID = IID_IErrorInfo;
-            int hr = Marshal.QueryInterface(externalComObject, ref errorInfoIID, out IntPtr comObject);
+            int hr = Marshal.QueryInterface(externalComObject, IID_IErrorInfo, out IntPtr comObject);
```

Comment:
Looks like this one is cross compiled for netstandard2.0, and so we need to continue using `ref` and to suppress the warning. It could be coupled with `AsRef` to avoid the copy downlevel (given we know it won't be mutated)